resources/views/livewire/users-table.blade.php

```diff
@@ -65,7 +65,7 @@
                 <td>{{ $user->name }}</td>
                 <td>
                     @if($user_profile = $user->profiles->first())
-                        <a href="{{ route('profiles.show', ['profile' => $user_profile]) }}">{{ $user->pea }}</a>
```

Comment:
The change in this file shouldn't be a part of this PR, since it's a part of the other one that's already been merged. See the following as one way to remove it: https://stackoverflow.com/questions/39459467/remove-a-modified-file-from-pull-request